### PR TITLE
routing: measure CreateFilter latency

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Add `MeasureFilterCreate` method to `Metrics` interface similar to existing `MeasureFilterRequest` and `MeasureFilterResponse`.

The alternative would be to use `MeasureSince` but then filter name would have to be embedded into metric key.

### CODE REVIEW

File: routing/datasource.go [hunk 1]
```diff
@@ -222,19 +222,24 @@ func createFilter(fr filters.Registry, def *eskip.Filter, cpm map[string]Predica
 		return nil, fmt.Errorf("filter %q not found", def.Name)
 	}
 
+	start := time.Now()
 	f, err := spec.CreateFilter(def.Args)
 	if err != nil {
```
[the remark this answers is not in the document]
Comment (AlexanderYastrebov) [reply routing/datasource.go]: Thanks, fixed.

**Comment by MustafaSaber** [comment]
should we update docs on the same PR?

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
> should we update docs on the same PR?

In this case I think we can do the docs later, because we should make sure we have useful docs in our operations doc.
We should write about it why would you should monitor this, so I think we should add a new paragraph to monitor some control plane parts.

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**